backend/coins/btc/electrum/client/client.go

```diff
@@ -128,58 +131,42 @@ func (client *ElectrumClient) RegisterOnConnectionStatusChangedEvent(onConnectio
 	})
 }
 
-// ServerVersion is returned by ServerVersion().
-type ServerVersion struct {
-	Version         string
-	ProtocolVersion *semver.SemVer
+// electrumxVersion is returned by serverVersion().
+type electrumxVersion struct {
+	software string
+	protocol *semver.SemVer
 }
 
-func (version *ServerVersion) String() string {
-	return fmt.Sprintf("%s;%s", version.Version, version.ProtocolVersion)
+func (v electrumxVersion) String() string {
+	return fmt.Sprintf("%s;%s", v.software, v.protocol)
 }
 
-// UnmarshalJSON implements the json.Unmarshaler interface.
-func (version *ServerVersion) UnmarshalJSON(b []byte) error {
-	slice := []string{}
-	if err := json.Unmarshal(b, &slice); err != nil {
-		return errp.WithContext(errp.Wrap(err, "Failed to unmarshal JSON"), errp.Context{"raw": string(b)})
+// serverVersion makes the "server.version" RPC call.
+// ElectrumX will reply with a success only once. Subsequent calls return
+// a "server.version already sent" error. Electrs doesn't enforce "only once"
```

Comment:
make an issue upstream at electrs?

Reply:
I'd rather make an upstream issue about implementing `server.features`... or send them a PR.

Reply:
then an upstream issue at electrumx? :joy: at least they should behave the same way, one way or another.

Reply:
ok, I sent them https://github.com/spesmilo/electrumx/pull/87